src/api/v1beta1/dynakube_status.go

```diff
@@ -29,14 +29,24 @@ type DynaKubeStatus struct {
 	// Conditions includes status about the current state of the instance
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 
-	ActiveGate   ActiveGateStatus   `json:"activeGate,omitempty"`
-	OneAgent     OneAgentStatus     `json:"oneAgent,omitempty"`
-	CodeModules  CodeModulesStatus  `json:"codeModules,omitempty"`
-	Synthetic    SyntheticStatus    `json:"synthetic,omitempty"`
+	// Observed state of ActiveGate
+	ActiveGate ActiveGateStatus `json:"activeGate,omitempty"`
+
+	// Observed state of OneAgent
+	OneAgent OneAgentStatus `json:"oneAgent,omitempty"`
+
+	// Observed state of Code Modules
+	CodeModules CodeModulesStatus `json:"codeModules,omitempty"`
+
+	// Observed state of Synthetic
+	Synthetic SyntheticStatus `json:"synthetic,omitempty"`
+
+	// Observed state of Dynatrace API
```

Comment:
nitpick: I don't think `observed state of x` is that correct, but I don't really have a better one

Reply:
Me neither... I got inspiration from the DynakubeStatus description
```
// DynaKubeStatus defines the observed state of DynaKube
```